jumanji/environments/logic/minesweeper/utils_test.py

```diff
@@ -75,11 +75,11 @@ def test_count_adjacent_mines(
     expected_count_adjacent_mines_result: int,
 ) -> None:
     """Test whether the mine counting function is working as expected"""
-    action_height, action_width = action
+    action_row, action_col = action
     assert (
         count_adjacent_mines(
             manual_start_state,
-            jnp.array([action_height, action_width], dtype=jnp.int32),
+            jnp.array([action_row, action_col], dtype=jnp.int32),
```

Comment:
Same question/remark here.
```suggestion
            action,
```